book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] @sanieni6 great work in adding the add_rental method, however for us to achieve the 'has-many' association between a person and book through the rental class, we need to pass `date, self, person` as  parameters when creating a new rental